fix: Update README.md, add instructions for: (#50)
colindean/fillertext: README.md

Contributing to FillerText
--------------------------
### Setting up the project

> 1. Create a fork of the project.

Reviewer: ```suggestion
1. [Create a fork](https://github.com/colindean/fillertext/fork) of the GitHub project repo.
```

2. `git clone` the forked repository to your local system.

> 3. Make sure that the Ruby version is `>= 2.6.0`.

Reviewer: ```suggestion
3. Make sure that the version of Ruby you're using meets the [minimum requirement in the gemspec](https://github.com/colindean/fillertext/blob/master/fillertext.gemspec#L15). 
```

Let's eliminate a place I need to bump versions yearly ;-)

> 4. Run `bundle install`.

Reviewer: ```suggestion
4. Run `bundle install` to install dependencies. It should tell you at this point if you don't have an adequate Ruby version installed.
5. Check basic sanity by running `bundle exec rake` to run tests and checks _before_ you make any changes.
6. Make your changes and commit them to a feature branch. If you're picking a particular GitHub issue, please put `gh-##-topic` in the branch where `##` is the ticket number and `topic` is a few words about it.
7. Push to your own repo and then submit a pull request to this one!
```
